Documentation/building/unix-instructions.md

```diff
@@ -13,31 +13,40 @@ For more information about the different options when building, run `build.sh -?
 
 ## Minimum Hardware Requirements
 - 2GB RAM
```

Comment:
Recommend adding `./` as "run `./build.sh -?` " to remind users that they will need it.